README.md

```diff
@@ -299,14 +299,17 @@ module.exports = {
 
 ```js
 // .eslintrc.mjs
-import tsResolver from 'eslint-import-resolver-typescript'
+import * as tsResolver from 'eslint-import-resolver-typescript'
 
 export default {
   settings: {
     'import-x/resolver': {
       name: 'tsResolver', // required, could be any string you like
       // enable: false, // optional, defaults to true
-      options: { someConfig: value }, // optional, options to pass to the resolver
+      // optional, options to pass to the resolver https://github.com/import-js/eslint-import-resolver-typescript#configuration
+      options: {
+        bun: true, // optional, resolve Bun modules, defaults to false
```

Comment:
Same as https://github.com/import-js/eslint-import-resolver-typescript/pull/404#discussion_r2011813366

Reply:
[my response](https://github.com/import-js/eslint-import-resolver-typescript/pull/404#discussion_r2011844477) from over there:

> yeah, I guess I'm thinking that there will be a lot of projects that don't want to switch to running ESLint with Bun (eg. just introducing some small Bun scripts in a larger project)
> 
> but happy to also include this - do you think that it would be enough to mention it in the comment? or should it have a paragraph outside of the code block?

Reply:
linked to the new section from import-js/eslint-import-resolver-typescript#404

- e9d54cf85053bb58b4562427a6425fd4b333a163